examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/0_Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<blockquote>With a single-line-of-code</blockquote><p>With a single line of code</p><p> </p><blockquote>This dramatically reduces the total runtime for many variational algorithms</blockquote><p>This reduces the total runtime for many variational algorithms by up to 10x. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>